### ISSUE TITLE
hugo v0.42.1 incompatibility with list.html

### ISSUE THREAD

**Issue by nhejazi** [ribice/kiss#37]
I've been using this theme for one of my personal Hugo-based websites. I just upgraded the submodule oI had of this theme from  d6a5676 to `master` and found that running `hugo build` via the [`blogdown` R package](https://github.com/rstudio/blogdown) (via `blogdown::build_site()`) is now broken, with error

```bash
Rscript -e "blogdown::build_site()"
Building sites … ERROR 2018/06/18 11:47:45 Error while rendering "taxonomy" in "": html/template:_default/list.html:1:12: no such template "theme/index.html"
```

In investigating this, I found that the source of the error is the presence of the style file `layouts/_default/list.html`. As far as I can tell, simply removing this file resolves the problem. This file has apparently been around for quite a while but I had not encountered any error caused by this when using previous versions of Hugo. Is this a known issue? If so, is the fix provided in #38 appropriate?

**Comment by tun** [comment]
I have the same issue and also I removed the line from `layouts/_default/list.html` but I think #38 is not the right solution because the build will lack of a tag/category list page.

**Comment by tun** [comment]
Just made a pull request #39 with just a path fix to make it work properly and without broken list pages.

**Comment by ribice** [comment]
Resolved with #39

**Comment by nhejazi** [comment]
Great! Thanks for fixing this with a better solution @tun, and thanks for quickly merging @ribice.
